lib/cmd/fhc/login.js

```diff
@@ -11,8 +11,7 @@ var crypto;
 var targets = require("./targets.js");
 var fhreq = require("../../utils/request");
 var ngui = require("../common/ngui");
-var proxyConfig = require('../../utils/proxy');
-var request = proxyConfig.configureProxy(require('request'), fhc.config.get('proxy'));
+var request = require('request');   // don't set proxy or other configured defaults here, since the config may not be loaded when this module is
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingComment)